intellij-idea/.idea/runConfigurations/ServerBoot.xml

```diff
@@ -8,7 +8,7 @@
       </pattern>
     </extension>
     <option name="SPRING_BOOT_MAIN_CLASS" value="de.metas.ServerBoot" />
-    <option name="VM_PARAMETERS" value="-DPropertyFile=$ProjectFileDir$\metasfresh.properties -DTraceFile=true -Xms256M -Xmx4G -Drun-headless=false -DdevelopmentWorkspaceDir=$ProjectFileDir$/metasfresh/backend" />
+    <option name="VM_PARAMETERS" value="-DPropertyFile=$ProjectFileDir$\metasfresh.properties -DTraceFile=true -Xms256M -Xmx736M -Drun-headless=false -DdevelopmentWorkspaceDir=$ProjectFileDir$/metasfresh/backend" />
```

Comment:
Is there a problem if we keep `-Xmx4G`?

Reply:
i experienced two things
* `-Xmx736M` works. I created1K sales orders (1 line each) and the same number of shipments on my laptop with this setting
* `-Xmx4G` is *a lot*. it cann make sense in production, but for DEV-using it's oversized
* my old notebook has 8GB RAM all in all and as i wrote, its working, *if* we don't give webapi and app 4GB each